OKTA-1216425 - Add JavaScript tab to OIE upgrade API/SDK guide (#6288)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/javascript/upgrade.md

> The Identity Engine SDK gives your applications more flexibility to create and configure user access experiences. The SDK's architecture shifts control of application configurations and authorization policies to the org administrator. This removes the need for code-level updates. The SDK's recursive method calls also support dynamic Identity Engine policies. These policies may require different access or authorization steps for different users.

Reviewer: ```suggestion
The Identity Engine SDK gives your apps more flexibility to create and configure user access experiences. The SDK's architecture shifts control of app configurations and authorization policies to the org admin. This removes the need for code-level updates. The SDK's recursive method calls also support dynamic Identity Engine policies. These policies may require different access or authorization steps for different users.
```

Reviewer: By org admin, do you mean an Okta user who's assigned the org admin role? Or do you mean it in a more general sense? If it's general, maybe just use "Okta admin".

Author: Meant it in the general sense (whoever in the org has rights to configure app/policy settings), not the specific "Org Admin" role — went with "org admin" per your suggested edit. Flag if you'd rather have "Okta admin" to avoid any read as the specific role name.

Reviewer: Can we use "Okta admin" so it's more general?

Author: Done — changed "org admin" to "Okta admin" in `upgrade.md` (pushed in 725577090). Thanks!

> If your browser-based application uses the Classic Engine Authentication SDK for custom authentication, the Identity Engine SDK is the ideal upgrade path. This move benefits your app with Identity Engine's features and configuration flexibility. It also removes your app's dependency on third-party cookies. The Classic Engine `session.setCookieAndRedirect` method requires third-party cookie access to retrieve tokens, and many browsers now block that access by default. The Identity Engine SDK returns tokens directly in the transaction response instead.

Reviewer: ```suggestion
If your browser-based app uses the Classic Engine Authentication SDK for custom authentication, the Identity Engine SDK is the ideal upgrade path. This move benefits your app with Identity Engine's features and configuration flexibility. It also removes your app's dependency on third-party cookies. The Classic Engine `session.setCookieAndRedirect` method requires third-party cookie access to retrieve tokens, and many browsers now block that access by default. The Identity Engine SDK returns tokens directly in the transaction response instead.
```

> The upgrade process also stays non-disruptive and iterative over time. Your Classic Engine Authentication SDK application code works with an Identity Engine configured org, and you can test and migrate use cases incrementally. See the following three examples for further information on upgrading your application code:

Reviewer: ```suggestion
The upgrade process also stays non-disruptive and iterative over time. Your Classic Engine Authentication SDK app code works with an Identity Engine configured org, and you can test and migrate use cases incrementally. See the following three examples for further information on upgrading your app code:
```
